discovery-aws-api/src/main/resources/reference.conf

```diff
@@ -2,6 +2,15 @@
 # Akka Service Discovery AWS Config                  #
 ######################################################
 
+ec2-tag-based-blocking-dispatcher {
+  type = Dispatcher
+  executor = "thread-pool-executor"
+  thread-pool-executor {
+    fixed-pool-size = 4
+  }
+  throughput = 1
+}
```

Comment:
Perhaps better to use the default shared akka.actor.default-blocking-io-dispatcher

Reply:
got it, will fix, actually totally forgot that there's a default shared blocking io dispatcher ;-)